mule/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: mule
+    version: 1.2
+
+source:
+    fn: mule.tar.gz
+    url: https://github.com/SciTools/mule/releases/download/vn1.2/mule.tar.gz
+
+build:
+    number: 0
+    skip: True # [py3k or win]
```

Comment:
Why? It should be skipped whener p3k is true, and also when win is true. If you used and it would skip only on p3k running on windows.